### PR TITLE
Brings the captain's safe off the wall, safes now save their contents

### LINKED ISSUE #80588: deconstructing the wall the spare id safe is on drops the spare
Reporting client version: 515.1623

## Issue Summary

destroying the wall under the spare safe
destroys the safe and drops the spare

## Round ID:
[221164](https://scrubby.melonmesa.com/round/221164)

## Testmerges:
- [Save json savefiles to disk in pretty print form](https://www.github.com/tgstation/tgstation/pull/80512)
- [(Testmerge Ready) New Antagonist: Tiger Cooperative Fanatic](https://www.github.com/tgstation/tgstation/pull/80313)
- [[TM ONLY] Remove 'natural' beheading](https://www.github.com/tgstation/tgstation/pull/80439)

## Reproduction:

1. destroy wall supporting safe
2. profit

### PR DESCRIPTION
**Pull request by JohnFulpWillard**
## About The Pull Request

I originally was gonna make the captain's spare ID safe float in the air if the wall under it was taken down, but it looked poor and was going against the vision for wall items (and go against the wallening) so my alternative proposition is this, taking the safe off the wall.
It's now a golden safe (like the one in the vault) but is interacted the exact same, it's now just a thing on the floor rather than being on a wall.
I'm not a spriter so I didn't give it a custom icon but if anyone wants to they can feel free to add one, just a golden version of the regular safe felt kinda eh.

I also added a wallframe version of the secure safe for when it is taken down. It will conserve its contents and be permanently locked until put back up. This doesn't apply to the new captain safe since it isn't a wall item.

## Why It's Good For The Game

Closes https://github.com/tgstation/tgstation/issues/80588
Prevents people from cheesing the spare and/or it being too easy to destroy.

## Changelog

:cl:
fix: The captain's safe is no longer on the wall, therefore cannot be cheesed by breaking the wall it sits on.
fix: Tearing down a wall that a safe is on now drops the safe with its contents, rather than dropping the contents onto the floor. The safe's contents cannot be interacted with while it's not on a wall.
/:cl:

### CODE REVIEW

**Comment by Sadboysuss** [comment]
Would thermite or emag break it open though? There should still be some way to open the safe other than using the correct code

**Comment by JohnFulpWillard** [comment]
I’m not touching its destruction methods, they’ve always existed and will continue to, this is just removing a recent exploit to bypass it

**Comment by Admiral-Gigaton** [comment]
and this is the exact same as before, just a resprite and a moved position? cant crack it with a stethoscope?

**Comment by DaCoolBoss** [comment]
Why big safe instead of floor safe? Floor safe could even reuse the old sprite.

**Comment by rageguy505** [comment]
Is there a update paths for it

**Review by MrMelbert — CHANGES_REQUESTED** [review]
Workable idea but nawwwwwww this needs a new sprite, a big ass safe for an ID card which is probably smaller than 4 inches wide is just absurd

image

At the very least you can shrink the safe sprite, that doesn't take spiriting ability to do

Also it would be nice to be able to sit this on top of tables

**Review by Wallemations — APPROVED** [review]

**Comment by JohnFulpWillard** [comment]
@MrMelbert looks like this now
image

**Comment by JohnFulpWillard** [comment]
> Is there a update paths for it

It's hard to make an update paths for this cause it's just removing directionals but it has to be manually moved since its now close to the comms console and on a table rather than a wall.

**Comment by MrMelbert** [comment]
> MrMelbert looks like this now

Better, can you give it an `anchored_tabletop_offset` so it doesn't hang so low?

**Comment by JohnFulpWillard** [comment]
image

**Comment by Thunder12345** [comment]
Did you just scale that down you madman

**Comment by DrDiasyl** [comment]
The color makes me puke.
Can you make it actually golden?

File: code/game/objects/structures/secure_safe.dm [hunk 1]
```diff
@@ -43,11 +74,14 @@ MAPPING_DIRECTIONAL_HELPERS(/obj/structure/secure_safe, 32)
 		It is made out of the same material as the station's Black Box and is designed to resist all conventional weaponry. \
 		There appears to be a small amount of surface corrosion. It doesn't look like it could withstand much of an explosion.\
 		It remains quite flush against the wall, and there only seems to be enough room to fit something as slim as an ID card."
```
Comment (san7890) [inline code/game/objects/structures/secure_safe.dm:76]: you need to update this description if you're not putting it on the wall anymore.

**Comment by JohnFulpWillard** [comment]
> The color makes me puke.
> Can you make it actually golden?

I didn't touch the colors, it's the same as it was as a wall item

**Comment by DrDiasyl** [comment]
> > The color makes me puke.
> > Can you make it actually golden?
> 
> I didn't touch the colors, it's the same as it was as a wall item

The wall safe looked golden because the safe sprite was white.
But this safe sprite is blue, and blue being forced to look golden is what making it look bad

**Comment by JohnFulpWillard** [comment]
I made it out of actual gold now and it takes the gold's color, i have no idea if this changed the colors at all
image

**Comment by Thunder12345** [comment]
I mean it's still pretty ugly but at least it's not green

**Comment by LemonInTheDark** [comment]
We're gonna have to sprite dirs for this aren't we

**Comment by JohnFulpWillard** [comment]
I mean, you'd only make it if it's for regular safes, captain safes are fine looking forward.

**Pull request merged**